lib/rules/jsx-indent.js

```diff
@@ -77,20 +75,16 @@ module.exports = {
      * Responsible for fixing the indentation issue fix
      * @param {ASTNode} node Node violating the indent rule
      * @param {Number} needed Expected indentation character count
-     * @param {Number} gotten Indentation character count in the actual node/code
      * @returns {Function} function to be executed by the fixer
      * @private
      */
-    function getFixerFunction(node, needed, gotten) {
-      if (needed > gotten) {
-        var spaces = indentChar.repeat(needed - gotten);
```

Comment:
just let me know which way you'd like. _to me_ it seems silly to bring in a lib since the join trick is short, simple, and I think fairly obvious what it does. 😏  aka left-pad